src/util/loanUtils.js

```diff
@@ -138,15 +138,15 @@ export function getDropdownPriceArray(
 		? build5DollarsPriceArray(parsedAmountLeft).slice(0, 28)
 		: buildPriceArray(parsedAmountLeft, minAmount).slice(0, 20);
 
-	const showHugeAmount = enableHugeAmount && parsedAmountLeft > 500;
+	const showHugeAmount = isLoggedIn && parsedAmountLeft > 500;
```

Comment:
I forgot that we only showed the large amounts when logged in, makes sense to keep that in place.